docker-compose.yml

```diff
@@ -37,7 +37,6 @@ services:
       args:
         - PIP_INDEX_URL=${PIP_INDEX_URL}
         - PIP_REQUIREMENT=./requirements/dev.txt
-        - PRODSEC_EMAIL=${PRODSEC_EMAIL}
```

Comment:
It's fine to remove the label from this image, but we'll need to set this env var in OpenShift since one of our views has a "contact info" address that uses this value.

Reply:
Right, let's set it in the new deployment configs then.